lang/lang_en_hu.txt

```diff
@@ -1,1403 +1,1403 @@
+"Hatso old.[um]"
 
 #MSG_UNLOAD_FILAMENT_REPEAT c=20 r=4
 "Please unload the filament first, then repeat this action."
-"\x00"
+"Kerlek eloszor vedd ki a filamentet, majd probalkozz ujra."
 
 #MSG_CHECK_IR_CONNECTION c=20 r=4
 "Please check the IR sensor connection, unload filament if present."
-"\x00"
+"Nezd meg az IR szenzor csatlakoz., vedd ki a filam., ha bent van."
 
 #MSG_RECOVERING_PRINT c=20
 "Recovering print"
-"\x00"
+"Nyomt. visszaallit"
 
 #MSG_REMOVE_OLD_FILAMENT c=20 r=5
 "Remove old filament and press the knob to start loading new filament."
-"\x00"
+"Vedd ki a regi filamentet, majd nyomd meg a gombot az uj filament betoltesehez."
 
 #MSG_CALIBRATE_BED_RESET c=18
 "Reset XYZ calibr."
-"\x00"
+"XYZ kal. nullazas"
 
 #MSG_RESET c=14
 "Reset"
-"\x00"
+"Ujrainditas"
 
 #MSG_RESUME_PRINT c=18
 "Resume print"
-"\x00"
+"Nyomt. folytatasa"
 
 #MSG_RESUMING_PRINT c=20
 "Resuming print"
-"\x00"
+"Nyomtatas folytatasa"
 
 #MSG_BED_CORRECTION_RIGHT c=14
 "Right side[um]"
-"\x00"
+"Jobb old.[um]"
 
 #MSG_RPI_PORT c=13
 "RPi port"
-"\x00"
+"RPi port"
 
 #MSG_WIZARD_RERUN c=20 r=7
 "Running Wizard will delete current calibration results and start from the beginning. Continue?"
-"\x00"
+"A Varazslo futtatasa nullazza a jelenlegi kalibracios ertekeket es elolrol fog mindent kezdeni. Folytatod?"
 
 #MSG_SD_CARD c=8
 "SD card"
-"\x00"
+"SDkartya"
 
 #MSG_RIGHT c=10
 "Right"
-"\x00"
+"Jobb"
 
 #MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=20 r=3
 "Searching bed calibration point"
-"\x00"
+"Kalibracios pont keresese az asztalon"
 
 #MSG_LANGUAGE_SELECT c=18
 "Select language"
-"\x00"
+"Valassz nyelvet"
 
 #MSG_SELFTEST_OK c=20
 "Self test OK"
-"\x00"
+"Ondiagnosztika OK"
 
 #MSG_SELFTEST_START c=20
 "Self test start"
-"\x00"
+"Ondiagnosztika indul"
 
 #MSG_SELFTEST c=18
 "Selftest"
-"\x00"
+"Ondiagnosztika"
 
 #MSG_SELFTEST_ERROR c=20
 "Selftest error!"
-"\x00"
+"Ondiagnosztika hiba!"
 
 #MSG_SELFTEST_FAILED c=20
 "Selftest failed"
-"\x00"
+"Ondiag. sikertelen"
 
 #MSG_FORCE_SELFTEST c=20 r=8
 "Selftest will be run to calibrate accurate sensorless rehoming."
-"\x00"
+"A pontos szenzor nelkuli homing erdekeben lefuttatom az ondiagnosztikat."
 
 #MSG_SEL_PREHEAT_TEMP c=20 r=6
 "Select nozzle preheat temperature which matches your material."
-"\x00"
+"Valaszd ki a fuvoka homersekletet, amelyik megfelel az altalad hasznalt anyaghoz ajanlott homersekletnek."
 
 #MSG_SET_TEMPERATURE c=20
 "Set temperature:"
-"\x00"
+"Homerseklet beall.:"
 
 #MSG_SETTINGS c=18
 "Settings"
-"\x00"
+"Beallitasok"
 
 #MSG_SHOW_END_STOPS c=18
 "Show end stops"
-"\x00"
+"Vegallaskapcsolok"
 
 #MSG_FILE_CNT c=20 r=6
 "Some files will not be sorted. Max. No. of files in 1 folder for sorting is 100."
-"\x00"
+"Nehany fajl nem lesz rendezve. Legfeljebb 100 fajlt lehet rendezni egy konyvtaron belul."
 
 #MSG_SORT c=7
 "Sort"
-"\x00"
+"Rendez"
 
 #MSG_NONE c=8
 "None"
-"\x00"
+"Nincs"
 
 #MSG_SORT_TIME c=8
 "Time"
-"\x00"
+"Ido"
 
 #MSG_SEVERE_SKEW c=14
 "Severe skew"
-"\x00"
+"NagyMerol.hiba"
```

Comment:
@AttilaSVK it ugyan az mint "Súlyos ferdeség"

Reply:
Ugyanaz a véleményem, mint fentebb.